ember/app/ui/components/identity-members/template.hbs

```diff
@@ -26,7 +26,9 @@
             <div class="uk-text-meta">
               {{#each member.roles as |role index|}}
                 <span class={{if role.inactive "text-strike-through"}}>
-                  {{role.title}}{{if (not (eq index (sub member.roles.length 1))) ","}}
+                  {{#if role.title}}
+                    {{role.title}}{{if (not (eq index (sub member.roles.length 1))) ","}}
```

Comment:
maybe I'm too dumb but I find this confusing to read. If there is an easy way to refactor this and make it more readable it might be worth it.

Reply:
Changed to append comma in js